tests/test_service.py

```diff
@@ -141,7 +141,7 @@ def test_start_item(self, rp_service):
                                      'startTime': '1.2.3.', 'launchUuid': 111,
                                      'type': 'STORY', 'parameters': None,
                                      'hasStats': True,
-                                     'code_ref': 'path_for_script'},
+                                     'code_ref': None},
```

Comment:
codeRef.

Patching sys.argv at line 124 is not required anymore.
rp_service fixture already returns an instance of the ReportPortalService class. At line 131 you should write like this:

         rp_service.start_test_item(name="name",
                                                  ....)